tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -259,13 +279,82 @@ static int regularizeLLVM() {
   return 0;
 }
 
+static int parseSPVExtOption(
+    SPIRV::TranslatorOpts::ExtensionsStatusMap &ExtensionsStatus) {
+  // Map name -> id for known extensions
+  std::map<std::string, ExtensionID> ExtensionNamesMap;
+#define _STRINGIFY(X) #X
+#define STRINGIFY(X) _STRINGIFY(X)
+#define EXT(X) ExtensionNamesMap[STRINGIFY(X)] = ExtensionID::X;
+#include "LLVMSPIRVExtensions.inc"
+#undef EXT
+#undef STRINGIFY
+#undef _STRINGIFY
+
+  // Set the initial state
+  // During SPIR-V consumption, assume that any known extension is allowed
+  // During SPIR-V generation, assume that any known extension is disallowed
+  // During convertation to/from SPIR-V text representation, assume that any
```

Comment:
convertation -> conversion?

Reply:
And while you're here, probably worth ending sentences with `.` .

Reply:
Fixed